packages/fakta-omsorgen-for/src/components/SokerinfoTable.tsx

```diff
@@ -31,27 +30,31 @@ const SokerinfoTable: FunctionComponent<SokerinfoTableProps> = ({ header, forhol
           <span className={styles.tableheader}>
             {/* TODO: Riktig src ihht skisse. src for ikke alle forhold oppfylt? */}
             <Image src={checkSvg} />
-            <span>{header}</span>
+            <span>{intl.formatMessage({ id: header })}</span>
           </span>
         </>,
       ]}
       allowFormattedHeader
       noHover
     >
-      {forhold.map(forholdet => (
-        <TableRow key={forholdet.forholdskode}>
-          <TableColumn className={styles['sokerinfo--firstcol']}>{forholdet.forholdstekst}</TableColumn>
-          <TableColumn className="bold">
-            {forholdet.erOppfylt
-              ? intl.formatMessage({ id: 'SokerinfoTable.Ja' })
-              : intl.formatMessage({ id: 'SokerinfoTable.Nei' })}
-          </TableColumn>
-          <TableColumn>
-            {/* TODO: link css. lage generell ekstern link? */}
-            <EksternLink to={forholdet.link.to} text={forholdet.link.text} />
-          </TableColumn>
-        </TableRow>
-      ))}
+      {forhold
+        .filter(forholdet => typeof forholdet.erOppfylt === 'boolean')
```

Comment:
Hva skjer om et forhold ikke er oppfylt? Går vi automatisk til avslag?

Reply:
dersom harOmsorgenFor fra responsen er null skal vi samtidig få et aksjonspunkt der saksbehandler må bestemme om søker har omsorgen, samt begrunne dette. Jeg skal legge til radioknapper for dette valget nå